.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env
```

Comment:
what notes are you refering to @scullenBitwise

Reply:
These are the notes from above:

> We want to make sure none of your .env-type files are pushed to GitHub. Please add this filename to your .gitignore if it's not in there already and investigate how to make Git "forget" previously tracked files so that you can preserve the file while preventing it from being tracked/added to GH. Let me know if you have any questions on this.